utils-test/src/main/kotlin/Utils.kt

```diff
@@ -21,6 +21,8 @@ package com.here.ort.utils.test
 
 import java.io.File
 
+val ANALYZER_ROOT = File("").toPath()
```

Comment:
Can we get rid of this in favor of assigning a default value of

    File(".")

or

    File(System.getProperty("user.dir"))

to the `analyzerRoot` parameter?

Reply:
I was just checking how we did this before afeac51dd83ac22c78d71a4a8c684c6457b0a82b again: We should simply use `projectDir` in the respective tests.

Reply:
I thought about this, but it requires many more code changes than using a constant.